apps/api/src/app/modules/project/services/project.service.ts

```diff
@@ -28,6 +28,7 @@ export class ProjectService extends CrudService<Project, ProjectCreateDtoV, Proj
       const releaseData = {
```

Comment:
extract to `formatReleaseCreateData(project: Project): ProjectCreateDto`

```suggestion
      const releaseCreateData = formatReleaseCreateData(project);
```